docs/deployment.md

```diff
@@ -0,0 +1,237 @@
+then you would set `DJANGO_SETTINGS_MODULE=drunc_ui.settings.deployment`. An example
+settings file containing the minimum required configuration is given below. Also see the
+Django [settings reference documentation].
+
+[settings reference documentation]: https://docs.djangoproject.com/en/5.1/ref/settings/
+
+```python
+# extend the production settings by importing them
+import os
+from drunc_ui.settings._production import *
+
+ALLOWED_HOSTS = ["hostname.for.access.com"]
+# Admins will receive emails in event of system errors
+ADMINS = [("Chris Cave-Ayland", "c.cave-ayland@imperial.ac.uk")]
+
+# email settings
+# the SMTP backend is configured in _production.py, you can specify something else but
+# the below example settings assume you don't
+# review the other possible settings https://docs.djangoproject.com/en/5.1/ref/settings/#email-backend
+EMAIL_HOST = "smtp.host.domainname"
+SERVER_EMAIL = "noreply@host.com"
+DEFAULT_FROM_EMAIL = "noreply@host.com"
+
+# database settings - will depend on your choice in deployment
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "mydatabase",
+        "USER": "mydatabaseuser",
+        "PASSWORD": os.environ["DATABASE_PASSWORD"],
+        "HOST": "127.0.0.1",
+        "PORT": "5432",
+    }
+}
+
+# a basic example logging configuration with environment variable support
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+        },
+    },
+    "root": {
+        "handlers": ["console"],
+        "level": "WARNING",
+    },
+    "loggers": {
+        "django": {
+            "handlers": ["console"],
+            "level": os.getenv("DJANGO_LOG_LEVEL", "INFO"),
+            "propagate": False,
+        },
+    },
+}
+```
+
+### Environment Variables
+
+The web app supports (and, in the case of sensitive values, requires) some settings via
+environment variable. These are listed below:
+
+- DJANGO_SETTINGS_MODULE - as discussed above
+- SECRET_KEY - sensitive value used for cryptographic purposes
+- PROCESS_MANAGER_URL - host and port information for the process manager
+- CSC_URL - host and port information for the connectivity server
+- CSC_SESSION - name of the active drunc session
+
+There may be additional environment variables to set depending on your deployment
+settings file.
+
+### WSGI Server
+
+Django apps should be deployed using a production capable WSGI server. The [Django docs
+on WSGI] cover a number of options but gunicorn is recommended as it is already a
+dependency of `drunc`. If you use anything else will have to install it into the
+software environment e.g. `poetry run python pip install uwsgi`. Make sure to use the
+selected WSGI server when launching the web app and to configure an appropriate number
+of threads and processes for handling incoming requests.
```

Comment:
Maybe include an example command for running the app with gunicorn here? Like you did for installing uwsgi